dominion/frontend/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+
+//import { createMuiTheme } from "@material-ui/core/styles";
+import CssBaseline from "@material-ui/core/CssBaseline";
+
+import { URLS } from "../src/component/DocumentHead/PageHeads";
+import Home from "./pages/home";
+import Country from "./pages/country";
+import NotFound from "./pages/NotFound";
+
+function App() {
+  return (
+    <CssBaseline>
+      <BrowserRouter>
+        <Switch>
+          <Route exact path={URLS.HOME} component={Home} />
+          <Route path={URLS.COUNTRY} component={Country} />
+          <Route path={URLS.NOT_FOUND} component={NotFound} />
+          />
+        </Switch>
+      </BrowserRouter>
+    </CssBaseline>
+  );
+}
+
+export default App;
```

Comment:
Add:

 + Theme, and
 + Scroll to top.

Use sensors.AFRICA repo as a guide.

Reply:
Hi @kilemensi Adding theme has a story on pivotal. Could we include that in another PR? We can also include Scroll on top on that PR as well

Reply:
* The _contents_ of the theme itself yes, they need their own story @nerdlet but the infrastructure should be there. It's part of setting up Material UI I think.
 * Personally I also this Scroll to top is basic infrastructure/setup type of work.